litestar/typing.py

```diff
@@ -424,6 +424,12 @@ def is_typeddict_type(self) -> bool:
 
         return is_typeddict(self.origin or self.annotation)
 
+    @property
+    def type_(self) -> Any:
```

Comment:
I'm not sure if this is the best name for this field. I'm open to suggestions for a better name or keeping this is also fine by me.

Reply:
How is this different from `self.origin`? `origin` already has all the wrapper removed, and `origin` can't be `None` (afaik), so the right side of this `or` will never be triggered.

Reply:
`origin` is `None` for non-generic types.

Reply:
Yes.. Totally missed that. Sorry :upside_down_face: